packages/admin/cms-admin/src/pages/pageTree/PageLabel.tsx

```diff
@@ -11,16 +12,31 @@ import { usePageTreeContext } from "./usePageTreeContext";
 interface PageLabelProps {
     page: PageTreePage;
     disabled?: boolean;
+    isEditable?: boolean;
     onClick?: (e: React.MouseEvent) => void;
```

Comment:
Why can't we remove this from the label altogether and only use the RowClickContainer?

Reply:
Mainly because the container is just an overlay without any content. We would need an aria-label on the container to do that which is problematic because PageLabel is responsible for deciding what the label is. 

Maybe it's possible to move the Container inside the PageLabel but I have to try that.

Reply:
Or we could improve the API of PageLabel by adding a variant Prop and depending on that decide the necessary props (like onClick)